corehq/apps/dump_reload/tests/test_couch_dump_load.py

```diff
@@ -137,7 +136,7 @@ def _make_doc(doc_type, domain):
 
     def test_multimedia(self):
         from corehq.apps.hqmedia.models import CommCareAudio, CommCareImage, CommCareVideo
-        image_path = os.path.join('corehq', 'apps', 'hqwebapp', 'static', 'hqwebapp', 'images', 'commcare-hq-logo.png')
+        image_path = 'corehq/apps/hqwebapp/static/hqwebapp/images/commcare-hq-logo.png'
```

Comment:
Would that break for devs using Windows? I am not sure.

Reply:
I think it would still work on Windows, and I'm pretty sure we do not support running tests natively on Windows.